.gitignore

```diff
@@ -18,4 +18,4 @@
 **.env.production.local
 **.idea
 
-**npm-debug.log*
+**npm-debug.log*
```

Comment:
Hmm now it seems like you're getting rid of new lines at the end of some files. If this is unintentional can you put them back?